.github/scripts/update-sources.sh

```diff
@@ -7,23 +7,32 @@ function update_repository() {
     REPOSITORY=$2
     BRANCH=$3
     if [[ ! -d $DIRECTORY ]]; then
-        git clone $REPOSITORY -b $BRANCH --single-branch --depth 1 $DIRECTORY
+        git clone $REPOSITORY -b $BRANCH $DIRECTORY
```

Comment:
Theese changes are here only for convenience of the local development and are submitted in a separate PR https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/pull/232.